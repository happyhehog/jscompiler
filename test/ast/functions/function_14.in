function test() { "42" + 42; }