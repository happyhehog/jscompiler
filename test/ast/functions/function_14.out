ProgramNode (1:0, 1:29)
    FunctionDeclarationNode (1:0, 1:29)
    --> name:
        IdentifierNode (1:0, 1:29) : { name: test }
    --> body:
        BinaryExpressionNode (1:18, 1:25) { operator: + }
        --> left operand:
            LiteralNode (1:18, 1:18) : { value: "42" }
        --> right operand:
            LiteralNode (1:25, 1:25) : { value: 42 }
